Marlin/Marlin.ino

```diff
@@ -42,6 +42,8 @@
     #include <LiquidTWI2.h>
   #elif ENABLED(DOGLCD)
     #include <U8glib.h> // library for graphics LCD by Oli Kraus (https://code.google.com/p/u8glib/)
+  #elif ENABLED(LCM1602)
+    #include <LiquidCrystal.h> 
```

Comment:
Following the pattern of some of the other panel settings, should this include all the items named in `ultralcd_implementation_hitachi_HD44780.h` below? Then it would include all of these:
```cpp
  #elif ENABLED(LCM1602)
    #include <Wire.h>
    #include <LCD.h>
    #include <LiquidCrystal_I2C.h>
    #include <LiquidCrystal.h> 
```
If the code here is actually what you want, then you can probably just drop the special case entirely, because it's the same as the default.